action-install.sh

```diff
@@ -2,8 +2,8 @@
 
 set -e
 
-rime_version=1.8.5
-rime_git_hash=08dd95f
+rime_version=1.10.0
+rime_git_hash=295cb2a
```

Comment:
this commit https://github.com/rime/librime/commit/295cb2ab68f89ee9d3237c7d4b8033bda3f3b635

plugins used as shared libraries are only working correctly after https://github.com/rime/librime/commit/a79187966ee0328c3635a5c5f6ecb4cf1e3463ee

let's hold the change and update `rime_version` to the next librime tag.